jax/interpreters/partial_eval.py

```diff
@@ -1437,10 +1483,13 @@ def process_call(self, call_primitive, f, tracers, params):
     new_params = dict(params, call_jaxpr=convert_constvars_jaxpr(jaxpr))
     update_params = call_param_updaters.get(call_primitive)
     if update_params:
-      # TODO(mattjj): update donated_invars to work with axis size variables
-      new_params = update_params(new_params, [True] * len(tracers))
+      new_params = update_params(new_params, [True] * len(tracers),
+                                 len(consts) + len(dim_tracers))
     eqn = new_jaxpr_eqn([*constvars, *invars], outvars,
                         call_primitive, new_params, source_info)
+    # TODO TODO remove
```

Comment:
Is this comment intentional and meant to outlast this change?

Reply:
Whoops, thanks for the catch.